tests/e2e/virt_backup_restore_suite_test.go

```diff
@@ -223,6 +231,73 @@ var _ = Describe("VM backup and restore tests", Ordered, func() {
 			},
 		}, nil),
 
+		Entry("no-application CSI+datamover backup and restore, powered-off CirrOS VM", Label("virt"), VmBackupRestoreCase{
+			Template:   "./sample-applications/virtual-machines/cirros-test/cirros-test.yaml",
+			InitDelay:  2 * time.Minute,
+			PowerState: "Stopped",
+			BackupRestoreCase: BackupRestoreCase{
+				Namespace:         "cirros-test",
+				Name:              "cirros-test",
+				SkipVerifyLogs:    true,
+				BackupRestoreType: lib.CSIDataMover,
+				BackupTimeout:     20 * time.Minute,
+				PreBackupVerify:   vmPoweredOff("cirros-test", "cirros-test"),
+			},
+			RestoreErr: errors.New("error to expose snapshot: error to wait target PVC consumed"),
```

Comment:
why are we adding a test that is expected to fail in 1.4 branch?

Reply:
I probably should have marked this "do not merge", or taken this one test out instead. But it looks like it's the only one consistently failing on 1.4, I am still looking into why.